resume.md

```diff
@@ -1,49 +1,47 @@
+* **Languages**: Python
+* **Design**: Figma, Adobe Illustrator, Procreate, Photoshop, Final Cut Pro
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
-
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
-
-## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+**Graphic Designer**: Innovative Design | Berkeley CA *Aug 2020 - Present*
+*	Created stylescapes in a branding project
+*	designed product mockups for small businesses
 
-## AWARDS/ACKNOWLEDGEMENTS
-* **COLLEGE Alumni Scholarship Recipient** - Alumni Scholarship Committee... *Month 20XX*
-* **1st Place Best Hack** - Hackathon Event... *Month 20XX*
+**Design Coordinator**: Associated Students of the University of California | Berkeley CA *August 2020 - Present*
+*	Created the ASUC brand book for design
+*	In the process of designing ASUC merch
+
+**Co-Founder**: CreAItivity | Bay Area CA *Jul 2019 - Present*
+* Youth-led non-profit seeking to educate underrepresented youth about AI
+*	Hosted a COVID-19 Challenge, 5 AI related workshops, and published 2 issues of tech-zines
+* (https://www.creaitivity.org/)
+
+**Public Health Researcher**: ING (startup) | San Francisco CA *August 2020 - Present*
+*	Researched and evaluated 800 Bay Area restaurants with ING’s healthy criteria
+*	Constructed the data library of evaluated restaurants and updated the ING criteria list
 
-## Events/Conferences
-* **Cal Hacks 7.0 (remote)** - Berkeley, CA ... *Month 20XX*
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+**Illustration Artist**: "Made in Chinatown" Exhibit | San Francisco CA *Dec 2019 - Apr 2020*
+*	Created 30 digital illustrations for promoting small businesses in San Francisco Chinatown
```

Comment:
Be specific - what small businesses?  Mom and Pop shops with over 50+ years of residency?  Folks who do not have a presence on Social media?  People who do not speak english / technical experience on the internet?  Which platforms did you post to?